### ISSUE TITLE
[Docs] Add more example about Yarn workspaces foreach --include pattern

### ISSUE THREAD

**Issue by harrytran998** [yarnpkg/berry#2849]
When I read [this doc](https://yarnpkg.com/cli/workspaces/foreach), I don't see any example about --include pattern.

I try with some experimental like this:
1. `yarn workspaces foreach -p -i -v --include @project/(ui|core) run build`
2. `yarn workspaces foreach -p -i -v --include [@project/core, project/ui] run build,`
3. ... or anything I can try!

So hope your team can includes that <3

**Comment by jbreckmckye** [comment]
Yeah this confused me for a while. In the end I had to mess around adding debug statements to a de-minified version of the plugin, just to figure out what was going on!

Basically there's a `micromatch` matcher that gets applied to the idents of your workspaces. Your ident is based on the `name` field of the workspace `package.json`.

e.g. if I have a package.json with `name: @feature/cool-feature` I can use

```
yarn workspaces foreach --include '@features/*' --verbose run test:unit
```
